src/plugins/CssProcessorPlugin.ts

```diff
@@ -12,15 +22,25 @@ export default class CssProcessorPlugin implements ConfigPlugin {
       let createRule;
       const rules = [];
       const postCssLoader = requireModule.probe('postcss-loader');
+      const useDefaultPostCss: boolean = spin.options.useDefaultPostCss || false;
```

Comment:
There are cases it may fail if we only look for the file in one path for example `<project>/src`.
If we use `import` to import css from third party library that exists in `node_modules` or in case of Lerna project from one of the `packages`, it expects `postcss.confg.js` file exists otherwise builds will throw errors. 

Also, if the config file exists and for some reason, the third party library is using a different name than the default name, the build fails again. 

I think we need to dig little deep to find out the correct way to do it.